javaparser-core/src/main/java/com/github/javaparser/JavadocParser.java

```diff
@@ -57,9 +60,15 @@ public static Javadoc parse(String commentContent) {
             blockLines = Collections.emptyList();
         } else {
             descriptionText = trimRight(String.join("\n", cleanLines.subList(0, indexOfFirstBlockTag)));
-            blockLines = cleanLines.subList(indexOfFirstBlockTag, cleanLines.size()).stream()
-                    .filter(Utils.STRING_NOT_EMPTY)
-                    .collect(Collectors.toList());
+            String tagBlock = cleanLines.subList(indexOfFirstBlockTag, cleanLines.size())
```

Comment:
I do the combining and splitting because the combined block makes it easier to split according to the "javadoc rules". When working with each line on its own, I think I would have needed a weird looping, handling the backtracking etc. I hope my comments make it clearer!